code/control/DeployPlanDispatcher.php

```diff
@@ -290,6 +284,12 @@ protected function getGitPrevDeploys($project) {
 	 * @return SS_HTTPResponse
 	 */
 	protected function getAPIResponse($output, $statusCode) {
+		$secToken = $this->getSecurityToken();
+		$secToken->reset();
```

Comment:
Follow up question about resetting then: if we are not submitting the token as part of GET, should we then not reset the token on GET handlers in the backend? I'm slightly worried about concurrency: could this blow up if a GET request is followed by a POST in quick succession?